components/Tools/ValidityChecker/ResultsPanels.vue

```diff
@@ -137,15 +138,13 @@
     children_array,
     max_retry,
   ) {
-    const params = new FormData()
-    params.append("geode_object", object)
-    params.append("filename", filename)
-    params.append("test", test)
-
-    const route = `${route_prefix}/inspect_file`
+    const params = {
```

Comment:
If the variables name are matching, I don't think you have to do the name mapping